events/wc_society_events.txt

```diff
@@ -173,3 +173,297 @@ character_event = {
 		set_character_flag = ask_for_ancient_mark_cooldown_flag
 	}
 }
+
+#Write down whispers
+character_event = {
+	id = WCCSO.10200
+	desc = EVTDESC_WCCSO.10200
+	picture = GFX_evt_madness
+    border = GFX_event_normal_frame_religion
+
+    is_triggered_only = yes
+
+    trigger = { can_use_society_power_write_whispers_trigger = yes }
+
+    option = {
+    	name = EVTOPTA_WCCSO.10200
+    	add_character_modifier = {
+    		modifier = writing_whispers
+			years = 2
+    	}
+		if = {
+			limit = {
+				NOT = { piety = 500 }
+			}
+			piety = -250
+			set_character_flag = whispers_quality_1
+		}
+		else_if = {
+			limit = {
+				piety = 500
+				NOT = { piety = 750 }
+			}
+			piety = -500
+			set_character_flag = whispers_quality_2
+		}
+		else_if = {
+			limit = {
+				piety = 750
+				NOT = { piety = 1000 }
+			}
+			piety = -750
+			set_character_flag = whispers_quality_3
+		}
+		else = {
+			piety = -1000
+			set_character_flag = whispers_quality_4
+		}
```

Comment:
Spending all your piety is kinda not ok, so I suggest multiply required piety for tiers by factor of 3. E.G. for tier 4, require 3k piety, but spending only 1k.

Reply:
Or, if you don't want to increase requirements, move this effect to decision, so players will see how many piety they will spend before clicking on decision.

Reply:
moved to decision